server/src/metadata/tenant-initialisation/tenant-initialisation.service.ts

```diff
@@ -67,7 +67,7 @@ export class TenantInitialisationService {
    * @param dataSourceMetadataId
    * @param workspaceId
    */
-  private async createObjectsAndFieldsMetadata(
+  public async createObjectsAndFieldsMetadata(
```

Comment:
As mentioned in the PR, we don't want to call TenantInitialisation for migrations, we should move this responsibility somewhere else.

Reply:
I agree with that